datasets/case02/queries/0001_fetchObjectById.sql

```diff
@@ -7,5 +7,5 @@
-FROM   Object
+FROM   {DBTAG_A}Object
```

Comment:
What does the name DBTAG_A mean?
I think it's testing to see if it works to insert the database name here, right? If so, why put that in this test? (instead of adding a new test with some description of what's being tested)

Reply:
What they actually test is that queries with fully qualified table names work, which is how TSP normally makes requests. At some point in the future, it should also force the czar to copy the empty table from master to the local database.

Reply:
isee. FWIW I think there's more value add by creating a new test case that is named and/or states in the comments that it "tests that queries with fully qualified table names work". It this case it's implicit and prone to getting lost or not understood. At the very least I think a comment should be added to this test.